templates/app/assets/javascripts/solidus_starter_frontend.js

```diff
@@ -0,0 +1,16 @@
+// Placeholder manifest file.
+// the installer will append this file to the app vendored assets here: vendor/assets/javascripts/spree/frontend/all.js'
+
+// spree/frontend/all points to the
+// `vendor/assets/javascripts/spree/frontend/all.js` file generated by
+// `solidus:install`. See `setup_assets` at
+// https://github.com/solidusio/solidus/blob/master/core/lib/generators/solidus/install/install_generator.rb
+//= require spree/frontend/all
+
+//= require spree
```

Comment:
This seems the right place to get rid of this `spree` name as well. WDYT? Also, the description above could be probably changed to remove all the spree references.

Reply:
@kennyadsl Yep, I have just checked `core/app/assets/javascripts/spree.js.erb` in Solidus and I found that the Spree JS helpers there are either replaced or not used in Solidus Starter Frontend. I'll remove this require call.